### PR TITLE
Add Loading component to Assign volunteer search section, autofill temperature measured to current time

### LINKED ISSUE #1526: Add loading indication when a search in happening on Assign Volunteer section

### LINKED ISSUE #1531: Add autofill current time to `Temperature Measured at` field
Form link : [`/facility/01810c32-ec26-494a-9e65-40e5547a7746/patient/ed658453-49f6-4669-8267-1fff984a9475/consultation/23d8a8b9-0a94-4914-ab54-8c6dfb6b86d5/daily-rounds/738/update`](http://localhost:4000/facility/01810c32-ec26-494a-9e65-40e5547a7746/patient/ed658453-49f6-4669-8267-1fff984a9475/consultation/23d8a8b9-0a94-4914-ab54-8c6dfb6b86d5/daily-rounds/738/update)

### PR DESCRIPTION
**Pull request by abhinandanarya06**
### Solution Applied

1. Used [`material-ui Circular Progress Bar`](https://material-ui.com/api/circular-progress/)

5a58ea0c-ce36-40f0-984d-bfb84c039cf0

2. Fix #1531 

c68af83f-9bbd-4fb0-aee7-164270fd66bc

closes #1526 
closes #1531

### CODE REVIEW

**Comment by nihal467** [comment]
https://user-images.githubusercontent.com/57055998/125072757-9ad2dc00-e0d8-11eb-846e-b5f46893d3e4.mp4

test failed, when we are clicking anywhere in the input field, the droplist vanish

**Comment by ghost** [comment]
>  Untitled.Project.mp4 
> test failed, when we are clicking anywhere in the input field, the droplist vanish

Hmm ux was bad when modifying search query by placing cursor in between. Now fixed the problem. 
@nihal467 can you test it again?

**Comment by nihal467** [comment]
I am still facing the same problem

**Comment by ghost** [comment]
> I am still facing the same problem

I am not able to reproduce the problem. I think i didn't get exact problem. Can you elaborate?

**Comment by nihal467** [comment]
https://user-images.githubusercontent.com/57055998/125851324-515e34a2-d93c-4f59-a226-0941a43bc832.mp4

Even after inputting necessary fields, there is error poping up

**Comment by abhinandanarya06** [comment]
>  error.2.mp4 
> Even after inputting necessary fields, there is error poping up

Oh seems like api change. 😯 
@bodhish 
i have added `taken_at`, `rounds_type` field in the form, is it fine?
the form was missing these field since 5 days of [this latest commit file](https://github.com/coronasafe/care/blob/ce1c5fa290d0c5b1403a42485f59c16a7444823c/care/facility/models/daily_round.py#L130)

**Comment by abhinandanarya06** [comment]
image

Really very weird error is being encountered. 
This code is not found anywhere don't know why its showing error with random code 😅 
Also don't know why deploy preview is not showing atleast latest correct code 
@bodhish can you see what's wrong 😅

File: src/Components/Patient/DailyRounds.tsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ import {
   TELEMEDICINE_ACTIONS,
```
Comment (abhinandanarya06) [inline src/Components/Patient/DailyRounds.tsx:12]: ### latest working commit 053ee2e64c565ea1af30a14026a5f0578ce7d3b6

**Comment by ghost** [comment]
recreating the PR. May be deployment is not updated due to force pushes 😅

**Pull request closed without merging**